src/Maestro/Maestro.Web/Api/v2018_07_16/Controllers/ChannelsController.cs

```diff
@@ -294,5 +307,91 @@ public async Task<IActionResult> DeletePipelineFromChannel(int channelId, int pi
 
             return StatusCode((int)HttpStatusCode.OK);
         }
+
+        const int engLatestChannelId = 2;
+        const int eng3ChannelId = 344;
+
+        /// <summary>
+        ///   Get the dependency flow graph for the specified <see cref="Channel"/>
+        /// </summary>
+        /// <param name="channelId">The id of the <see cref="Channel"/></param>
+        /// <param name="includeDisabledSubscriptions">Include disabled subscriptions</param>
+        /// <param name="includedFrequencies">Frequencies to include</param>
+        /// <param name="includeBuildTimes"></param>
```

Comment:
>includeBuildTimes [](start = 25, length = 17)

What does this boolean control?  My guess is that it's something about some property in the returned object.  Since this documentation becomes swagger docs, they should be pretty detailed.